src/genai/text/moderation/moderation_service.py

```diff
@@ -76,6 +77,11 @@ def create(
             ApiNetworkException: In case of unhandled network error.
             ValidationError: In case of provided parameters are invalid.
         """
+        if implicit_hate is not None:
+            self._log_deprecation_warning("'implicit_hate' is deprecated, use 'social_bias' instead!")
```

Comment:
So we ignore them here, but not in generation service?

Reply:
Updated.